Pokemon App/Pokemon App/CardListCollectionView/PokemonCardListLoader.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PokemonCardListLoader.swift
+//  Pokemon App
+//
+//  Created by Nadia Darin on 06/11/22.
+//
+
+import Foundation
+import UIKit
+
+enum PokemonCardListResult {
+    case success([PokemonCard])
+    case failure(String)
+}
+
+class PokemonCardListLoader {
+    
+    func getMovieList(completion: @escaping (PokemonCardListResult) -> Void) {
+        let url = URL(string: "https://api.pokemontcg.io/v2/cards?pageSize=10")!
+        
+        
+        
+        URLSession.shared.dataTask(with: url) { (data, response, error) in
+            DispatchQueue.main.sync {
+                do {
+                    let movies = try self.transformJsonDataToMovieList(with: data!)
+                    completion(.success(movies))
```

Comment:
force unwrap `data!` bisa menyebabkan Force Close. Lebih baik dibuat 
```suggestion
            if let data = data,
               let response = response as? HTTPURLResponse,
               response.statusCode == 200 {
                let movies = try self.transformJsonDataToMovieList(with: data)
            } else {
                completion(.failure("Unexpected Data"))
            }
```